rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverter.groovy

```diff
@@ -82,6 +87,9 @@ class PackageNameConverter {
           release = parts.pop()
           version = parts.pop()
           name = parts.join("-")
+          qualifiedName = "${name}${versionDelimiter}${version}-${release}"
```

Comment:
We would need the versionDelimiter in the method. Having it as a part of the OsPackageName seems random. But passing it as a parameter makes more sense in that it keeps the OsPackageName agnostic to what PackageType it's derived from. I'll try that.